### PR TITLE
lint fixes for various tools

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/abricate/abricate_list.xml [hunk 1]
```diff
@@ -5,33 +5,28 @@
...
+    <expand macro="edam_ontology"/>
+    <expand macro="xrefs"/>
+    <expand macro="requirements"/>
+    <expand macro="version_command"/>
     <command detect_errors="exit_code"><![CDATA[
         abricate --list > '$report'
     ]]></command>
-
     <inputs>
+        <param name="hidden_input" type="hidden" label="This is to please our linter in this expeptional case"/>
```
Comment (bernt-matthias) [inline tools/abricate/abricate_list.xml:16]: I think it's better to leave the linter skip here

```suggestion
```

File: tools/barrnap/barrnap.xml [hunk 2]
```diff
@@ -52,24 +52,24 @@
     </outputs>
 
     <tests>
-        <test><!-- Test 1 -->
+        <test expect_num_outputs="1"><!-- Test 1 -->
```
Comment (bernt-matthias) [inline tools/barrnap/barrnap.xml:55]: Guess we can remove a linter rule from the skip file. 

We should bump and deploy this one (so maybe better put in a separate PR)?

**Review by bernt-matthias — COMMENTED** [review]
Can we remove `XMLOrder` now in all `.lint_skip` file?

Fine after vacation :)

File: tools/barrnap/barrnap.xml (hunk 2 above)
Comment (bgruening) [reply tools/barrnap/barrnap.xml:55]: Not sure a deployment is needed here? I would not deploy for a stricter test. Please feel free to merge and take it from here.

**Comment by bgruening** [comment]
@bernt-matthias what do you want me to do with this one?

File: tools/abricate/abricate.xml [hunk 3]
```diff
@@ -5,11 +5,10 @@
     <macros>
         <import>macros.xml</import>
     </macros>
-    <expand macro='xrefs'/>
-    <expand macro='edam_ontology'/>
-    <expand macro="requirements" />
-    <expand macro="version_command" />
-
+    <expand macro="edam_ontology"/>
```
Comment (bernt-matthias) [inline tools/abricate/abricate.xml:8]: If we fix a linter error we should remove it from the tool's `.lint_skip` file.

**Review by bernt-matthias — COMMENTED** [review]
Can you open a PR against IUC/main?

File: tools/abricate/abricate.xml (hunk 3 above)
Comment (bgruening) [reply tools/abricate/abricate.xml:8]: abcriacte didn't had a lint_skip file imho
